skins/cat17/faq-page/src/Faq.vue

```diff
@@ -12,14 +12,16 @@
  	<h2>{{ messages.page_title }}</h2>
  	<h5>{{ messages.page_subtitle }}</h5>
  	<div class="row">
- 	<div class="col-xs-12 col-sm-9">
+ 	<div class="col-xs-12 col-sm-8">
 	  	<search-bar :messageSearch="messages.search"></search-bar>
 		<div class="form-shadow-wrap">
-			<h2 v-if="!isPreview" class="title space-below underlined">{{ topicTitle }}</h2>
+			<h2 v-if="!isPreview" class="title">{{ topicTitle }}</h2>
 			<ul>
-				<li v-for="content in page" v-bind:class="[ isPreview ? 'preview' : 'topic', 'underlined' ]">
-					<question :content="content.question"></question>
-					<answer :hiddenContent="content.hidden_text" :visibleContent="content.visible_text" :messages="messages" :isPreview="isPreview"></answer>
+				<li v-for="(content, index) in page" v-bind:class="[ isPreview ? 'preview' : 'topic' ]">
```

Comment:
I suggest removing the `ul/li`tags and rendering the `question` elements directly - except if the lists strongly help with accessibility or you really want to convey the semantics that this is a list of questions.

If you render the questions directly, you can use the the `:key` attribute, as proposed in https://vuejs.org/v2/guide/list.html You can then drop `:idx` and  use `:key` instead, getting the additional benefit of faster list rendering.